manifests/slave.pp

```diff
@@ -49,7 +49,7 @@
 #   Not required.  Single string of whitespace-separated list of labels to be assigned for this slave.
 #
 # [*tool_locations*]
-#   Not required.  Single string of whitespace-separated list of tool locations to be defined on this slave. A tool location is specified as 'toolName:location'.
+#   Not required.  Array list of tool locations to be defined on this slave. A tool location is specified as 'toolName:location'.
```

Comment:
This does change an API which concerns me, it would be better to take an Array or the single String for the parameter and then prepare the appropriate data for the template